Meeting notes 2021-03-17 (#195)
solid/authorization-panel: meetings/2021-03-17.md

# 2021-03-17 Authorization Panel

Time: 10am ET / 15:00 CET (timezone shift)

https://meet.jit.si/solid-authorization

## Present
* [Sarven Capadisli](https://csarven.ca/#i)
* Justin Bingham
* Matthieu Bosquet
* Henry Story
* Elf Pavlik
* Dmitri Zagidulin

## Agenda

### Announcements

### Pull Requests
* [Meeting Notes for 2021-03-03](https://github.com/solid/authorization-panel/pull/186)
* [Omit solution-centric references](https://github.com/solid/authorization-panel/pull/173)

### Issues
* [Update references made to UCR URLs](https://github.com/solid/authorization-panel/issues/192)
    * Proposed Action: assignment

### Discussion

> * WAC, WAC(+) and ACP, can they coexist or are they mutually exclusive? (Asked by @elf-pavlik). 

Reviewer: ```suggestion
* WAC, WAC(+), and ACP: can they coexist or are they mutually exclusive? (Asked by @elf-pavlik). 
```

Here is material to help:
   - [Re-using Policies](https://github.com/solid/authorization-panel/issues/184) brings WAC+ and ACP closer together

>    -  Discuss allowing ACLs to be visible needed by both WAC+ and ACP:

Reviewer: ```suggestion
   -  Discuss allowing ACLs to be visible, as needed by both WAC+ and ACP:
```

      + [ACLs on ACLs](https://github.com/solid/authorization-panel/issues/189)

>       + [ACP allows restictions on ACRs](https://github.com/solid/authorization-panel/issues/151)

Reviewer: ```suggestion
      + [ACP allows restrictions on ACRs](https://github.com/solid/authorization-panel/issues/151)
```

>       + [PR 250 in protocol: reduce requirements on acl:Control to edit and create](https://github.com/solid/specification/pull/250)

Reviewer: ```suggestion
      + [PR 250 in protocol: reduce requirements on `acl:Control` to edit and create](https://github.com/solid/specification/pull/250)
```

  - Comparison Table:
      + [PR: Proposal Evaluation Table](https://github.com/solid/authorization-panel/pull/180)

>       + [PR 173: Omit Solution Centric references](https://github.com/solid/authorization-panel/pull/173)

Reviewer: ```suggestion
      + [PR 173: Omit Solution-Centric references](https://github.com/solid/authorization-panel/pull/173)
```

      + [WAC ACP Diff Story](https://github.com/solid/authorization-panel/pull/178)

## Actions

1. Previous meeting notes to be merged [Meeting Notes for 2021-03-03](https://github.com/solid/authorization-panel/pull/186) after the call waiting for a few more +1s

> 3. [Omit solution-centric references](https://github.com/solid/authorization-panel/pull/173)
> 4. Commit Elfs [PR: Proposal Evaluation Table](https://github.com/solid/authorization-panel/pull/180) after an addition of an intro text.
> 
> 
> 
> 
> 
> 

Reviewer: ```suggestion
2. [Omit solution-centric references](https://github.com/solid/authorization-panel/pull/173)
3. Commit Elf's [PR: Proposal Evaluation Table](https://github.com/solid/authorization-panel/pull/180) after addition of an intro text
```
